Api/Endpoints/Files.php

```diff
@@ -68,6 +68,9 @@ public function fetchAll($projectId)
      */
     public function upload($projectId, $body)
     {
+        // Requests that do not use the queue=true parameter are deprecated and will be unsupported
+        $body['queue'] = true;
```

Comment:
Forcing `queue` to be enabled my break stuff for clients. Didn't we have some transition period?

Reply:
Or library is forcing queue since `3.0.0`?

Reply:
This is a major version update (3.0.0). 2.* are deprecated, but you can use them. 3.0.0 is the version that works only in the new way (confirmed)